doc/arkode/guide/source/Mathematics.rst

```diff
@@ -562,6 +569,63 @@ conservation property :cite:p:`HaWa:06`,  SPRKStep employs a fixed time-step siz
 .. The `ark_kepler.c` example demonstrates an implementation of such controller. 
 
 
+.. _ARKODE.Mathematics.SPRKStep.Nonautonomous:
+
+Non-autonomous systems
+----------------------
+
+SPRKStep also supports non-autonomous separable Hamiltonian systems of the form
+
+.. math::
+   \dot{p} = f_1(q, t) = \frac{\partial V(q, t)}{\partial q}, \quad 
+   \dot{q} = f_2(p, t) = \frac{\partial T(p, t)}{\partial p}, 
+   \qquad p(t_0) = p_0,\quad q(t_0) = q_0,
+   :label: ARKODE_IVP_Hamiltonian_Time_Dependent
+  
+where
+
+.. math::
+   H(p, q, t) = T(p, t) + V(q, t)
+
+In this non-autonomous case, the Hamiltonian no longer yields a conserved quantity (such as energy)
+although other invariants may exist :cite:p:`Struckmeier:02`.
+However, symplectic methods still are benefical to use for the long-time integration of these systems.
+For example, they may accurately capture the bounded energy of the system :cite:p:``.
```

Comment:
Missing citation

Reply:
Fixed in the re-written section. I remove this non-autonomous section as I realized that with a minor tweak in TakeStep we do not actually need the extended state-space.  Instead, I generalized the whole section to nonautonomous.